### DISCUSSION TITLE
[Question] Link rulus of docsusaurus markdown

### DISCUSSION THREAD

**Discussion by Beacontownfc** [facebook/docusaurus#7738] [Q&A]
In the blue box of the following picture, how does docusaurus detect whether the path is relative path or absolute path?
![f652dee1be04d880f5ed1d977b167f6](https://user-images.githubusercontent.com/89081023/177898557-5b2020cc-10c9-4c3b-a19f-fb38004c1e5a.png)

**Comment by Josh-Cena** [comment]
A _relative path_ is a path that does not start with `/`; an absolute path is one that does.

**Reply by Beacontownfc** [reply]
But the above document says that the path start with "/" （`/otherFolder/doc4.md`） is also a relative path.

**Reply by Josh-Cena** [reply]
Ah, the link _is_ an absolute path, per se, but we will _resolve_ it relative to the site directory. An "absolute path" still has to be relative to something to be resolved.
